support-frontend/test/fixtures/TestCSRFComponents.scala

```diff
@@ -13,14 +13,7 @@ trait TestCSRFComponents {
 
   private lazy val appComponents = {
     val env = Environment.simple(new File("."))
-    val configuration = Configuration.load(env)
-    val context = ApplicationLoader.Context(
-      environment = env,
-      sourceMapper = None,
-      webCommands = new DefaultWebCommands(),
-      initialConfiguration = configuration,
-      lifecycle = new DefaultApplicationLifecycle()
-    )
+    val context = ApplicationLoader.Context.create(env)
```

Comment:
looks good - same as https://www.playframework.com/documentation/2.8.x/Migration27#Application-Loader-API-changes

Reply:
That's where I took it from 😅